src/nl/documentatie/10.woocommerce-v5.0.md

```diff
@@ -0,0 +1,329 @@
+# WooCommerce
+
+<Stack class="grid-flow-col auto-cols-max gap-1">
+    <GitHubShield repo="myparcelnl/woocommerce/" />
+    <GitHubShield repo="myparcelnl/woocommerce" link="issues" />
+    <GitHubShield repo="myparcelnl/woocommerce" link="pulls" />
+</Stack>
+
+## Introductie
+
+<MPImg src="/documentation/woocommerce/nl/myparcel_plugin_api-key.jpeg" alt="MyParcel plugin API key" />
```

Comment:
Dit is niet echt een intro he :P 

Je kunt even kijken naar de lap tekst die we posten op de alpha releases, die info zou hier zeker niet misstaan denk ik. En hoort dit plaatje niet hieronder bij het instellen van de api key?

Reply:
Nee, goed punt. Oude handleiding introductie was een opsomming van alle opties/mogelijkheden. Lap tekst onder alpha gaat niet echt over de opties / mogelijkheden van de MyParcel plugin. Afbeelding is inderdaad ter verduidelijking voor het instellen van de API key, niet duidelijk genoeg?